external-crates/move/crates/move-vm-runtime/src/tracing2/trace_macros.rs

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
I'm not a big fan of using these macros here, in part because of the definitional behavior of `cfg` inside of them, but also because I feel like this is all rather straightforward code to simply write.

Reply:
Good point! Will update :)